src/main/java/io/dropwizard/kafka/tracing/TracingFactory.java

```diff
@@ -50,7 +50,6 @@ public void setRemoteServiceName(final String remoteServiceName) {
         }
 
         return Optional.of(KafkaTracing.newBuilder(tracing)
-                .writeB3SingleFormat(writeB3SingleFormat)
```

Comment:
does this make sense to change as part of this PR?

Reply:
It is a no-op now, and is the default behavior as part of a brave update. So this is removing a deprecated method call that does nothing